### PR TITLE
Include client mTLS credentials in credentials.zip if available

### PR DESCRIPTION
**Pull request by simao**

### CODE REVIEW

File: ota-plus-web/app/com/advancedtelematic/api/CryptApi.scala [hunk 1]
```diff
@@ -111,6 +111,18 @@ class CryptApi(conf: Configuration, val apiExec: ApiClientExec)(implicit exec: E
       .execJson[DeviceRegistrationCredentials](apiExec)
   }
 
+  def downloadClientCredentials(accountName: String): Future[HttpEntity] = {
+    baseUri(s"/accounts/$accountName/credentials/client")
```
Comment (simao) [inline ota-plus-web/app/com/advancedtelematic/api/CryptApi.scala]: will probably need to be changed, to what?

File: ota-plus-web/test/com/advancedtelematic/controllers/ClientToolControllerSpec.scala [hunk 2]
```diff
@@ -196,13 +219,35 @@ class ClientToolControllerSpec extends PlaySpec with GuiceOneServerPerSuite with
...
+
+      status(result) mustBe OK
+      contentType(result).get mustBe "application/zip"
+
+      val zipContents = readZip(contentAsBytes(result))
+
+      zipContents.keys must contain("client_auth.p12")
+    }
+
+    "returns a zip archive containing without mTLS credentials if they exist do not exist" in {
```
Comment (jochenschneider) [inline ota-plus-web/test/com/advancedtelematic/controllers/ClientToolControllerSpec.scala]: wording

**Comment by houcros** [comment]
@simao it's been a while, credentials are in `user-profile` now 😄 So I will close here.
Is this still something we want? If so I can create a ticket to do this in `user-profile`.

**Comment by simao** [comment]
this was to include mtls credentials in credentials.zip but since we are not doing mtls anymore I guess this is obsolete.

**Pull request closed without merging**